tests/unit/nupic/research/temporal_memory_test.py

```diff
@@ -999,6 +999,27 @@ def testMapCellsToColumns(self):
     self.assertEqual(columnsForCells[99], set([399]))
 
 
+  def testMaxSegmentsPerCellGetter(self):
+    tm = TemporalMemory(
+      columnDimensions=[64,64],
+      cellsPerColumn=32,
+      maxSegmentsPerCell=200
+    )
+    self.assertEqual(tm.connections.maxSegmentsPerCell, tm.getMaxSegmentsPerCell())
+    self.assertEqual(tm.connections.maxSegmentsPerCell, 200)
+    self.assertEqual(tm.getMaxSegmentsPerCell(), 200)
+
+  def testMaxSynapsesPerSegmentGetter(self):
+      tm = TemporalMemory(
+        columnDimensions=[32,32],
+        cellsPerColumn=16,
+        maxSynapsesPerSegment=150
+      )
+      self.assertEqual(tm.connections.maxSynapsesPerSegment, 150)
+      self.assertEqual(tm.getMaxSynapsesPerSegment(), 150)
+      self.assertEqual(tm.connections.maxSynapsesPerSegment, tm.getMaxSynapsesPerSegment())
```

Comment:
Also remove this line, because it is testing something that you've already tested. I think you only need the one assertion: `self.assertEqual(tm.getMaxSynapsesPerSegment(), 150)`